conf.py

```diff
@@ -101,19 +101,21 @@
         "font-stack": "Inter, sans-serif",
         "color-sidebar-brand-text": "#4a4b57",
         "color-foreground-primary": "#333333",
-        "color-foreground-secondary": "#787885",
-        "color-foreground-muted": "#787885",
+        "color-foreground-secondary": "#747481",
+        "color-foreground-muted": "#747481",
         "color-foreground-border": "#e1e1e3",
         "color-background-primary": "#ffffff",
         "color-background-secondary": "#f7f7fa",
-        "color-content-foreground": "#787885",
+        "color-content-foreground": "#747481",
         "color-background-hover": "#c60443",
         "color-background-border": "#e1e1e3",
         "color-highlighted-background": "#1c1c2f",
-        "color-inline-code-background": "#787885",
+        "color-inline-code-background": "#747481",
         "color-sidebar-background": "#f7f7fa",
         "color-sidebar-background-border": "#e1e1e3",
         "color-sidebar-search-background": "#fff",
+        "sd-color-card-background": "#f7f7fa",
+        "sd-color-primary": "#4a4b57",
```

Comment:
This value ends up also used for dark mode. But for dark mode the contrast gets too low. We need to add something brighter and to add it for "dark_css_variables".
For example, when looking at the search input border, in the dark mode we use there `--color-foreground-border: #e1e1e3; `. For me that color  plays similar role to button border, that's why we can use it for the button as well.
<img width="802" alt="Screenshot 2022-08-03 at 12 02 40" src="https://user-images.githubusercontent.com/4600541/182581920-9cd6b58c-88a0-4ec0-a360-001400f54dd1.png">